tensorbay/client/version.py

```diff
@@ -582,3 +582,94 @@ def delete_tag(self, name: str) -> None:
         delete_data: Dict[str, Any] = {"name": name}
 
         self._client.open_api_do("DELETE", "tags", self._dataset_id, json=delete_data)
+
+
+class JobMixin:  # pylint: disable=too-many-public-methods
+    """A mixin class supporting asynchronous jobs.
+
+    Arguments:
```

Comment:
`Attributes`

Reply:
shall we put private attributes in docstring?